packages/config/config/devices/0x0460/qnsw-001P16.json

```diff
@@ -29,31 +28,58 @@
 			"maxNodes": 9
 		}
 	},
+
 	"paramInformation": [
 		{
 			"#": "1",
 			"label": "Sw1 Switch Type",
-			"description": "0",
+			"description": "This parameter defines how the Device should treat the switch (which type) connected to the SW (SW1) terminal.",
```

Comment:
We try to avoid parameter descriptions unless necessary for understanding what the parameter does, or to provide additional information like gaps in the allowed parameter range.
https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide?id=parameter-descriptions